integration_tests/mockApis/curiousApi.ts

```diff
@@ -316,6 +317,125 @@ const stubLearnerEducation = (prisonNumber = 'G6115VJ', page = 0): SuperAgentReq
     },
   })
 
+const stubLearnerEducationWithCoursesQualificationsInLast12Months = (
+  prisonNumber = 'G6115VJ',
+  page = 0,
+): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPathPattern: `/learnerEducation/${prisonNumber}`,
+      queryParameters: {
+        page: { equalTo: `${page}` },
+      },
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: {
+        content: [
+          {
+            prn: prisonNumber,
+            establishmentId: 'WDI',
+            establishmentName: 'WAKEFIELD (HMP)',
+            courseName: 'GCSE Maths',
+            courseCode: '246674',
+            isAccredited: true,
+            aimSequenceNumber: 1,
+            learningStartDate: moment().subtract(8, 'months').toDate(),
+            learningPlannedEndDate: moment().subtract(2, 'months').toDate(),
+            learningActualEndDate: moment().subtract(1, 'months').toDate(),
+            learnersAimType: 'Component learning aim within a programme',
+            miNotionalNVQLevelV2: 'Level 5',
+            sectorSubjectAreaTier1: 'Science and Mathematics',
+            sectorSubjectAreaTier2: 'Science',
+            occupationalIndicator: false,
+            accessHEIndicator: false,
+            keySkillsIndicator: false,
+            functionalSkillsIndicator: false,
+            gceIndicator: false,
+            gcsIndicator: false,
+            asLevelIndicator: false,
+            a2LevelIndicator: false,
+            qcfIndicator: false,
+            qcfDiplomaIndicator: false,
+            qcfCertificateIndicator: false,
+            lrsGLH: 0,
+            attendedGLH: 100,
+            actualGLH: 200,
+            outcome: 'No achievement',
+            outcomeGrade: null,
+            employmentOutcome: null,
+            withdrawalReasons: 'Other',
+            prisonWithdrawalReason: 'Significant ill health causing them to be unable to attend education',
+            completionStatus: 'The learner has withdrawn from the learning activities leading to the learning aim',
```

Comment:
Something is not right here ....
The tech notes in the story say this:

> **Completed courses**
> Filter courses from API response by `completionStatus === Completed`

So, I would expect this line in your stub to say 
```
completionStatus: 'Completed',
```

But because it doesn't ..... it's making me wonder how the code is even working? Does your implementation (in your previous PR) not filter on `completionStatus` ?

Reply:
The code I have added uses the result of `const completedInPrisonEducation = completedInPrisonEducationRecords(allInPrisonEducation)` 

The existing `inPrisonEducationRecordsResolver` : `completedInPrisonEducationRecords` should handle the `completionStatus` filtering.

Reply:
Hmmm, well I don't understand whats going on here then .... 🤔  .... how is this a "completed" course? 🤔

Reply:
We still need to get to the bottom of this - how and why is this course considered a "completed" course?